apps/web/lib/hooks/useMediaQuery.ts

```diff
@@ -1,3 +1,4 @@
+// lets refactor and move this into packages/lib/hooks/
 import { useState, useEffect } from "react";
 
 const useMediaQuery = (query: string) => {
```

Comment:
@zomars can you explain me why some lib/hooks are in /apps/web and others in packages/libs/hooks?

What is the decision making process of where the hook should go?

the useMediaQuery hook seems to be reusable in all of our apps so I am surprised to find it here.

I duplicated the code now because I am not sure if we want to refactor or keep it separate

Reply:
if we want to refactor, lets move this into packages and adjust the imports